trace-demo/values.yaml

```diff
@@ -0,0 +1,210 @@
+observability:
```

Comment:
Many of the values here are the same values as in the upstream chart. Can we just add our modifications?

Reply:
I'm in the progress of fixing it upstream, hope to get it sorted out soon: https://github.com/open-telemetry/opentelemetry-helm-charts/issues/438

For now we need to copy them